lib/public.js

```diff
@@ -36,11 +37,34 @@ class Public {
      */
     publicRequest({
         resource,
-        params
+        params,
+        post=false,
```

Comment:
Would it make sense to default to POST and make a note in the changelog? My understanding is that we are allowing GET requests for backwards compatibility and recommend using POST for all new implementations.

Reply:
Product has decided to keep it the way it was for now and make this an option, not the default functionality.